prompt.sh

```
@@ -1,32 +1,328 @@
+
+  # All dirty files (modified and untracked)
+  #git_dirty_count=$(cat "$gs_porc_file" | wc -l)
+  # Only modified files
+  #git_dirty_count=$(grep -c -v '^??' "$gs_porc_file")
+  # Only modified files which have not been staged.  The second grep hides staged [M]odified files, staged [A]dded files, staged [D]eletes and staged [R]enames.
+  # Whitelist: Hide things which we know mean cleanly staged.
+  #git_dirty_count=$(grep -v '^??' "$gs_porc_file" | grep -c -v '^[AMDR] ')
+  # Blacklist: Hide things which we think mean cleanly staged.
+  # That is anything with a space in the second column, or in the case of staged resolved merge conflicts, 'UU'.
+  git_dirty_count=$(grep -v '^??' "$gs_porc_file" | grep -c -v '^\([^ ?] \|UU\)')
+  if [[ "$git_dirty_count" > 0 ]]; then
```

Shellcheck (0.11.0)

[error] > is for string comparisons. Use -gt instead.

(SC2071)